tests/test_solvers/test_scipy_solver.py

```diff
@@ -9,6 +9,64 @@
 import numpy as np
 
 
+def mesh_for_testing(number_of_pts=0):
```

Comment:
could you use the default mesh in `shared.py` here?

Reply:
Ah yes, missed these when cleaning it up, thanks